test/e2e/storage/testsuites/selinuxmount.go

```diff
@@ -0,0 +1,232 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testsuites
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/component-base/featuregate"
+	"k8s.io/kubernetes/pkg/features"
+	"k8s.io/kubernetes/test/e2e/feature"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	e2evolume "k8s.io/kubernetes/test/e2e/framework/volume"
+	storageframework "k8s.io/kubernetes/test/e2e/storage/framework"
+	admissionapi "k8s.io/pod-security-admission/api"
+	"k8s.io/utils/ptr"
+)
+
+type seLinuxMountTestSuite struct {
+	tsInfo storageframework.TestSuiteInfo
+}
+
+var (
+	// Pod SELinux label used in the test pods.
+	seLinuxLabel = &v1.SELinuxOptions{Level: "s0:c0,c1"}
+	// The expected SELinux mount option, matching both Debian (system_u:object_r:svirt_lxc_net_t) and Fedore based distros (system_u:object_r:container_file_t)
+	seLinuxMountOptionRE = "context=\"system_u:object_r:[^:]*:s0:c0,c1\""
+)
+
+// InitCustomSELinuxMountTestSuite returns seLinuxMountTestSuite that implements TestSuite interface
+// using custom test patterns
+func InitCustomSELinuxMountTestSuite(patterns []storageframework.TestPattern) storageframework.TestSuite {
+	return &seLinuxMountTestSuite{
+		tsInfo: storageframework.TestSuiteInfo{
+			Name:         "seLinuxMount",
+			TestPatterns: patterns,
+			SupportedSizeRange: e2evolume.SizeRange{
+				Min: "1Mi",
+			},
+			TestTags: []interface{}{
+				feature.SELinux,
+				framework.WithFeatureGate(features.SELinuxMountReadWriteOncePod),
+				framework.WithFeatureGate(features.SELinuxChangePolicy),
+			},
```

Comment:
Why doesn't this include `SELinuxMount` featuregate btw?

Reply:
Ah, looks like these are enabled by default whereas other get tunrned-on selectively. hhmm